src/components/common/form/message-item-types/DoubleDateMoved.spec.jsx

```diff
@@ -0,0 +1,29 @@
+import { screen, fireEvent, waitFor } from '@testing-library/react';
+import { renderWithTimecardContext } from '../../../../test/helpers/TimecardContext';
+import SingleDateMoved from './DoubleDateMoved';
+
+describe('DoubleDateMoved', () => {
+  it('should render a summary message with two date links', async () => {
+    renderWithTimecardContext(
+      <SingleDateMoved
+        variables={['2022-09-21', '2022-09-22']}
+        setSummaryMessages={jest.fn()}
+        setIsAlertVisible={jest.fn()}
+      />
+    );
+
+    await waitFor(() => {
+      const message = screen.getByText(
+        'The time period starts on and finishes on'
+      );
+      expect(message).toBeTruthy();
+
+      const startDateLink = screen.getByRole('link', { name: '21 September' });
+      expect(startDateLink.pathname).toBe('/timecard/2022-09-21');
+
+      const endDateLink = screen.getByRole('link', { name: '22 September' });
+      expect(endDateLink.pathname).toBe('/timecard/2022-09-22');
+      fireEvent.click(endDateLink);
```

Comment:
Do we need to click the link after the expect statement?

Reply:
I realised after going through all the files that this is probably to clear the summary messages - could we reset them before each test or add a comment to this line instead?

Reply:
Updated the tests to move the link click to it's own test and make sure it's calling the expected function